src/qt/bitcoingui.cpp

```diff
@@ -681,6 +704,54 @@ void BitcoinGUI::openClicked()
     }
 }
 
+void BitcoinGUI::newWallet()
+{
+    QString file_name = QFileDialog::getSaveFileName(this, tr("New Wallet"));
```

Comment:
This still defaults to the directory that qt is running in. Can we change that to the wallet directory?